src/applications/static-pages/facilities/FacilityDetailWidget.jsx

```diff
@@ -21,29 +22,52 @@ export class FacilityDetailWidget extends React.Component {
     // Sort and compile facility hours into a list
     const hours = facilityDetail.attributes.hours;
     const builtHours = buildHours(hours, true);
+    const clinicalHours = builtHours.map((day, index) => {
+      const splitDay = day.split(': ');
+      const abbrvDay = splitDay[0];
+      const times = splitDay[1];
+
+      let el = '';
+      if (index === 0 || index === 5) {
+        el += `<ul class="vads-u-flex--1 va-c-facility-hours-list vads-u-margin-top--0 ${
+          index === 0
+            ? 'vads-u-margin-bottom--1 small-screen:vads-u-margin-bottom--0 vads-u-margin-right--3'
+            : 'vads-u-margin-bottom--0'
+        }">`;
+      }
+
+      el += `<li key=${index}>
+                <b class="abbrv-day">${abbrvDay}:</b> ${times}
+              </li>`;
+
+      if (index === 4 || index === 6) {
```

Comment:
I think it's possible to make this cleaner in idiomatic React, which would also not require the new `ReactHtmlParser` dependency. The handling of magic numbers for the indexes is going to be harder to maintain going forward.

Reply:
It looks like there are going to be two `<ul>` elements here. In that case, it would be easier to read if it were explicit:

```
const clinicalHours = builtHours.map((day, index) => {
   ...
   return <li>...</li>;
});
```

```
<ul className="vads-u-margin-bottom--1 ...">{clinicalHours.slice(0, 5)}</ul>
<ul className="vads-u-margin-bottom--0">{clinicalHours.slice(5, clinicalHours.length)}</ul>
```

But maybe declaring the significance of the index 5 by assigning it to a descriptive variable for context.

Reply:
how would you do it? you have 7 days in the week and you need to split the two columns by monday tuesday wednesday thursday friday in one column and saturday and sunday in the other. please let me know. i really struggled hard to get this working and it seemed impossible with the way the data is set up.